### ISSUE TITLE
Consider using a theme icon for the theme templates

### ISSUE THREAD

**Issue by juanfra** [WordPress/gutenberg#68142]
## What problem does this address?

There are different icons used to represent template categories—user and plugin templates each have their own. However, theme templates don’t have a unique icon. It also feels a bit redundant that the “All Templates” icon is the same as the one for theme templates.

<img width="1995" alt="Image" src="https://github.com/user-attachments/assets/bbe192e6-55cf-496d-8273-9b548be8e343" />

## What is your proposed solution?

It would feel more intuitive if the theme category had its own unique icon. While we already have an icon for plugins in the design system, there doesn’t seem to be one specifically for themes. One option could be to create a new icon for this purpose. Alternatively, we could use an existing icon from the system that better represents themes, like brush.

Pinging @WordPress/gutenberg-design

**Comment by jasmussen** [comment]
That seems valid. It needs a good icon design, one that feels conceptually related to the current icon. We have sidebar, footer, header versions of the "layout" icon already, that feel connected. What's a way to create a remix of the layout icon that's just for the active theme? Maybe a solid version rather than outline version?

**Comment by jameskoster** [comment]
Yeah I don't think we were able to settle on an icon in the current set that represents 'theme', which is why the `layout` icon was reused.

Conceptually themes are a bundle of design assets; templates, styles, fonts, graphics, etc. Maybe something that plays on that?

<img width="965" alt="Image" src="https://github.com/user-attachments/assets/0accb2ed-4a19-44c3-82c7-58d5203a0119" />

Or perhaps something more standard like a paint brush, echoing to the icon we find in wp-admin.

<img width="965" alt="Image" src="https://github.com/user-attachments/assets/42ae4849-4bcd-4b81-9176-0d4aad717464" />

**Comment by juanfra** [comment]
Thanks so much for looking into this!

I really like both directions. Personally, I feel the paintbrush icon could create more cohesion with what WordPress users are already familiar with, making it especially helpful for those transitioning to the site editor.

**Comment by jasmussen** [comment]
Of the two, I'd choose the masonry over the paintbrush. We already have a paintbrush in the verse icon, which leverages solids, suddenly introducing the outline style for painting accessories exits that metaphor a bit. That's not to say it's not a nice design, that was quite a good brush though.

**Comment by richtabor** [comment]
Are those icons helpful on the grid there anyhow?

**Comment by fcoveram** [comment]
I like the brush. It is consistent with the concept. The "Appearance" action in Admin and even the Theme team. I envision the masonry one for a new grid view, if the CSS inclusion keeps improving.

**Comment by jasmussen** [comment]
It's really a well designed brush. My hesitancy is that it feels like a different visual language than these similar utensils we have:

![Image](https://github.com/user-attachments/assets/8d0f5284-2ba7-4cc9-82ed-e071e44b3fd5)
![Image](https://github.com/user-attachments/assets/c20fe6a0-a676-4f61-b266-fa8ffbbfb90f)

**Comment by juanfra** [comment]
It’s a bit tricky because the design language of these icons (`verse` and `edit`) might feel out of place in the sidebar, where the other icons have more of an outlined look.

While we do have the `brush` icon as an option, it feels slightly disconnected from the rest in my opinion.

https://github.com/user-attachments/assets/7d1a7aca-8e51-4598-88fb-ad77744af375

**Comment by richtabor** [comment]
I actually lean towards no icons here. If they're not adding communicative value, I question if they're needed.

**Comment by juanfra** [comment]
> I actually lean towards no icons here. If they're not adding communicative value, I question if they're needed.

I’d keep the icons in the sidebar to maintain consistency with the other sections. I also think they add some helpful context about what the “filter” represents. That said, I don’t have strong opinions about the icons for the listed elements on the right.

**Comment by annezazu** [comment]
Looping back on this and wanted to share what we currently have:

<img width="296" height="403" alt="Image" src="https://github.com/user-attachments/assets/d644a239-12b3-4d5b-a55b-97d4ca526078" />

I'm going to close this out as a result but cc @ellatrix  who is working hard on various template improvements as something to carry forward in the work there as I do think it's helpful to distinguish between theme templates and user created templates.

**Comment by juanfra** [comment]
Thanks for the update, Anne! 

I've been following the recent template improvements, and they’re looking great. I still think the icon choice might not be the best fit for theme templates, since it currently uses the same one as the general “Templates” section. It feels like introducing a distinct "theme" icon (like we were discussing, similar to how plugins have their own) could help make that distinction clearer.

Happy to revisit this anytime if it becomes relevant in the future.
